pootle/templates/staticpages/_body.html

```diff
@@ -9,7 +9,14 @@
   {% endblocktrans %}
   </div>
   {% endif %}
-  <h1 class="title">{{ page.title }}</h1>
+  <h1 class="title">
+    {{ page.title }}
+    {% if user.is_superuser %}
```

Comment:
By type of announcement:

- Project announcement: Can be edited by global admins and project managers
- Language announcement: Can be edited by global admins and language managers
- TP announcement: Can be edited by global admins and TP managers

As commented above we don't seem to have any permission to tell if a user is a project manager or language manager.